### ISSUE TITLE
Add option to automatically copy screenshot when saved using (Ctrl+S)

### ISSUE THREAD

**Issue by faizul726** [flameshot-org/flameshot#3690]
### Feature Description

This is something that I have seen in LightShot. It's useful in scenario like, you take a screenshot for future usage but also want send it to a friend right away.

In that case, automatically copying the screenshot to clipboard is really helpful.

**Comment by awwaiid** [comment]
What I do is to config it for the opposite -- set the `save image after copy` option and then use `ctrl+c` instead of `ctrl+s` and the result is the same, you get a file AND the image on the clipboard.

**Comment by faizul726** [comment]
@awwaiid good idea but what if someone wants to just copy a screenshot not save it?
